velox/dwio/common/ColumnVisitors.h

```diff
@@ -137,6 +137,64 @@ class ExtractToGenericHook {
   ValueHook* hook_;
 };
 
+class ExtractStringDictionaryToGenericHook {
```

Comment:
Can you move this back to the original location?  I think you no longer to change it right?